### ISSUE TITLE
Need to switch from v1beta1/poddisruptionbudgets to v1/poddisruptionbudgets

### ISSUE THREAD

**Issue by jarlebh** [cloudnative-pg/cloudnative-pg#772]
The OpenShift cluster complains about the v1beta1/poddisruptionbudgets, can you please update to use the v1 instead ?

_Deprecated API that will be removed in the next EUS version is being used. Removing the workload that is using the policy.v1beta1/poddisruptionbudgets API might be necessary for a successful upgrade to the next EUS cluster version. Refer to `oc get apirequestcounts poddisruptionbudgets.v1beta1.policy -o yaml` to identify the workload._

**Comment by gbartolini** [comment]
Support for Kubernetes 1.25 has already been developed and will be included in version 1.18, which is expected for the end of October.

https://github.com/cloudnative-pg/cloudnative-pg/commit/a64eda701984bff6736d75b361890895f43df4e0

FYI, as a Community (the CloudNativePG community) we have decided not to support OpenShift at this stage and concentrate on standard Kubernetes only. If you need OpenShift support, you need to look at third party vendors (the company I work for, for example, offers support on OpenShift).

**Comment by onedr0p** [comment]
Looks like this was released in 1.17.1 @gbartolini ?

**Comment by onedr0p** [comment]
Wait, I see the [commits](https://github.com/cloudnative-pg/cloudnative-pg/commit/a64eda701984bff6736d75b361890895f43df4e0) were included in the 1.17.1 release but the operator is still throwing

```
{"level":"error","ts":1665166607.357674,"msg":"pkg/mod/k8s.io/client-go@v0.24.4/tools/cache/reflector.go:167: Failed to watch *v1beta1.PodDisruptionBudget: failed to list *v1beta1.PodDisruptionBudget: the server could not find the requested resource\n","stacktrace":"k8s.io/client-go/tools/cache.(*Reflector).Run.func1\n\tpkg/mod/k8s.io/client-go@v0.24.4/tools/cache/reflector.go:222\nk8s.io/apimachinery/pkg/util/wait.BackoffUntil.func1\n\tpkg/mod/k8s.io/apimachinery@v0.24.4/pkg/util/wait/wait.go:155\nk8s.io/apimachinery/pkg/util/wait.BackoffUntil\n\tpkg/mod/k8s.io/apimachinery@v0.24.4/pkg/util/wait/wait.go:156\nk8s.io/client-go/tools/cache.(*Reflector).Run\n\tpkg/mod/k8s.io/client-go@v0.24.4/tools/cache/reflector.go:220\nk8s.io/apimachinery/pkg/util/wait.(*Group).StartWithChannel.func1\n\tpkg/mod/k8s.io/apimachinery@v0.24.4/pkg/util/wait/wait.go:56\nk8s.io/apimachinery/pkg/util/wait.(*Group).Start.func1\n\tpkg/mod/k8s.io/apimachinery@v0.24.4/pkg/util/wait/wait.go:73"}
```

**Comment by coolguy1771** [comment]
I'm encountering the same issue, it looks like something was missed when the patch to support 1.25 was added.

**Comment by mnencia** [comment]
The support for 1.25 is only present on the main branch and will be globally available with version 1.18. We decided not to include it in 1.17.1 because it also drops support for 1.19 and 1.20, and it is inappropriate to do that in a patch release.

**Comment by sxd** [comment]
It's on version 1.18.0, closing
